pkg/cloud/aws/actuators/scope.go

```diff
@@ -70,12 +72,17 @@ func NewScope(params ScopeParams) (*Scope, error) {
 		clusterClient = params.Client.Clusters(params.Cluster.Namespace)
 	}
 
+	if params.Logger == nil {
+		params.Logger = klogr.New().WithName("default-logger")
+	}
+
 	return &Scope{
 		AWSClients:    params.AWSClients,
 		Cluster:       params.Cluster,
 		ClusterClient: clusterClient,
 		ClusterConfig: clusterConfig,
 		ClusterStatus: clusterStatus,
+		Logger:        params.Logger.WithName(params.Cluster.APIVersion).WithName(params.Cluster.Namespace).WithName(params.Cluster.Name),
```

Comment:
How does chaining `WithName` work here? It'd be useful to form a path here with proper prefixes like in @detiber's other comment

Reply:
Chaining adds `/` between each .WithName